Makefile

```diff
@@ -1,14 +1,14 @@
 .PHONY: all build clean lint static
-MODULE:=Illinois_midPoint
-PACKAGE:=illinoismidpoint
+MODULE:=illinois_app
+PACKAGE:=app
 
 SRCS_DIR:=src/ph$(MODULE)
 TSCS_DIR:=tests
 SOAR_SRCS:=$(shell find $(SRCS_DIR) -type f)
 SRCS:=$(shell find $(SRCS_DIR) -name '*.py')
 TSCS:=$(shell find $(TSCS_DIR) -name '*.py')
-TAG_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
-HASH_FILES:=$(addprefix $(SRCS_DIR)/, app.py)
+VERSION_FILES:=$(addprefix $(SRCS_DIR)/, $(PACKAGE).json app.py)
+GITHUB_DEPLOYED:=$(shell date -u +%FT%X.%6NZ)
```

Comment:
Would BUILD_TIME be a better name?

Reply:
These are the names we used in the other projects. I'm down with updating the names here, this is where to get the names right so they move to other projects.

Reply:
Done.